decidim-conferences/app/helpers/decidim/conferences/conference_program_helper.rb

```diff
@@ -11,8 +11,10 @@ def meetings_for_day(component, day, user)
 
         meetings_by_time = {}
         meetings.each do |meeting|
-          meetings_by_time[start_time: meeting.start_time, end_time: meeting.end_time] ||= []
-          meetings_by_time[start_time: meeting.start_time, end_time: meeting.end_time] << { meeting: }
+          key = { start_time: meeting.start_time, end_time: meeting.end_time }
```

Comment:
In this case, is happening strange and the `meetings_by_time` is not built as we expected it in ruby 3.3.

Reply:
In Ruby 3.3 , without modifying the code, the unmodified version of the code will generate the below data structure, then it will raise a `NoMethodError`: 
```
{[]=>{
  :start_time=>Wed, 24 Jul 2024 13:47:40.777211000 UTC +00:00, 
  :end_time=>Wed, 24 Jul 2024 15:47:40.777211000 UTC +00:00
}}
```

The data structure needed is : 
```
{{
  :start_time=>Wed, 24 Jul 2024 13:51:39.367047000 UTC +00:00,
   :end_time=>Wed, 24 Jul 2024 15:51:39.367047000 UTC +00:00
}=> [{:meeting=> }, {:meeting=> }]
}
```

Reply:
Ah ok, thanks for the clarification!